test/e2e/go.mod

```diff
@@ -5,8 +5,8 @@ go 1.13
 require (
 	github.com/onsi/ginkgo v1.10.3
 	github.com/onsi/gomega v1.7.1
-	k8s.io/api v0.0.0-20191121015604-11707872ac1c
-	k8s.io/apimachinery v0.0.0-20191121015412-41065c7a8c2a
+	k8s.io/api v0.17.0
+	k8s.io/apimachinery v0.17.0
```

Comment:
These shouldn't be updates yet

Reply:
These were a result of `go mod tidy`. Added replace for those packages to pin versions.